src/azure-cli/azure/cli/command_modules/appservice/_constants.py

```diff
@@ -287,28 +287,28 @@ def __init__(self):
         }
     ],
     'linux': [
-        { "displayName": "DOTNETCORE|2.1" },
-        { "displayName": "DOTNETCORE|3.1" },
-        { "displayName": "NODE|12-lts" },
-        { "displayName": "NODE|10-lts" },
-        { "displayName": "NODE|10.1" },
-        { "displayName": "NODE|10.10" },
-        { "displayName": "NODE|10.12" },
-        { "displayName": "NODE|10.14" },
-        { "displayName": "NODE|10.16" },
-        { "displayName": "NODE|12.9" },
-        { "displayName": "JAVA|8-jre8" },
-        { "displayName": "JAVA|11-java11" },
-        { "displayName": "TOMCAT|8.5-jre8" },
-        { "displayName": "TOMCAT|9.0-jre8" },
-        { "displayName": "TOMCAT|8.5-java11" },
-        { "displayName": "TOMCAT|9.0-java11" },
-        { "displayName": "PHP|7.2" },
-        { "displayName": "PHP|7.3" },
-        { "displayName": "PYTHON|3.8" },
-        { "displayName": "PYTHON|3.7" },
-        { "displayName": "PYTHON|3.6" },
-        { "displayName": "RUBY|2.5.5" },
-        { "displayName": "RUBY|2.6.2" }
+        {"displayName": "DOTNETCORE|2.1"},
```

Comment:
Also consider moving this to a .JSON file? not needed right away but something to consider.